pkg/registry/registry/repository/storage/storage.go

```diff
@@ -119,6 +127,9 @@ type REST struct {
 	privilegedUsername string
 	harborClient       *harbor.APIClient
 	registryClient     *registryinternalclient.RegistryClient
+	distributionClient *client.Repository
+	// tokenPrivateKey    libtrust.PrivateKey
+	// repoEndpoint       string
```

Comment:
if you don't need L131-L132, pleae remove

Reply:
done